gcivil-nyu-org/fall2019-cs-gy-6063-team-debug_entities
# Name is displayed more than once in matching stack bug fix

### Description

User is displayed more than once in matching stack bug fix.

### Testing Directions

```
python manage.py migrate
python manage.py pull_seatgeek_data
python manage.py make_users
python manage.py runserver
```
Pull code from develop and try this scenario:

1. Steps to reproduce the bugs, Login as a user and mark interested/going to more than once event (more than 2 events user is interested in and more than 2 events user is going to)
2. Make more than 2-3 users with the above features.
3. See the matching stack for the events marked, the number of events user is going to is the number of times user's image is displayed on matching stack and same with interested. If the matching stack is in Interested events, user is displayed as many times as the user marked himself as going to and viceversa

Pull code from this branch and try the above steps once again
You should see that now the user is populated only once. 

Make sure squads is working fine as well. Make sure to test squads after this.
### Miscellaneous Comments

Please do thorough testing. Try every scenario possible.

### Checklist
